projects/step-frontend/src/lib/modules/execution/components/alt-execution-analytics/alt-execution-analytics.component.html

```diff
@@ -1 +1,7 @@
-<step-execution-page [execution]="(_state.execution$ | async)!" />
+@if (_state.execution$ | async; as execution) {
+  <step-execution-page
+    [execution]="execution!"
+    [initialTimeRange]="(_state.timeRangeChange$ | async) || undefined"
```

Comment:
@lucianbaciu Is this `|| undefined` necessary here?

Reply:
yes, the async returns model | null, which does not match the signature, so I convert it to undefined